sourced/ml/transformers/indexer.py

```diff
@@ -69,6 +75,11 @@ def calculate_value_to_index(self, rdd: RDD):
             raise RuntimeError("Number of distinct values is zero.")
         self._value_to_index = {d: i for i, d in enumerate(values)}
 
+    def save_index(self, path):
+        self._log.info("Saving the index to %s", self.path)
```

Comment:
`self.path` is invalid. This means that this method is not covered with tests :)

Reply:
hehe got me there, gonna add one